src/app/pipes/track.pipe.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { TrackPipe } from './track.pipe';
+
+describe('TrackPipe', () => {
+  it('create an instance', () => {
+    const pipe = new TrackPipe();
+    expect(pipe).toBeTruthy();
```

Comment:
Nothing important, I just love this "toBeTruthy" method name

Reply:
It's the default autogenerated test to check if the module is initialized correctly in the app. That's actually autogenerated code haha.